addons/im_livechat/models/res_users.py

```diff
@@ -10,12 +10,14 @@ class Users(models.Model):
     """
     _inherit = 'res.users'
 
-    livechat_username = fields.Char("Livechat Username", help="This username will be used as your name in the livechat channels.")
+    livechat_username = fields.Char(related='res_users_settings_id.livechat_username', readonly=False)
+    livechat_lang_ids = fields.Many2many(related='res_users_settings_id.livechat_lang_ids', readonly=False)
+    has_access_livechat = fields.Boolean(compute='_compute_has_access_livechat', string='Has access to Livechat', store=False, readonly=True)
 
     @property
     def SELF_READABLE_FIELDS(self):
-        return super().SELF_READABLE_FIELDS + ['livechat_username']
+        return super().SELF_READABLE_FIELDS + ['livechat_username', 'livechat_lang_ids', 'has_access_livechat']
 
-    @property
-    def SELF_WRITEABLE_FIELDS(self):
-        return super().SELF_WRITEABLE_FIELDS + ['livechat_username']
```

Comment:
How comes it is no longer necessary to have access for `SELF_WRITEABLE_FIELDS` but still for `SELF_READABLE_FIELDS`?